.github/workflows/test_codecov.yml

```diff
@@ -0,0 +1,41 @@
+name: CI test and coverage
+
+on:
+  pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    name: Run tests for ${{ matrix.python-version }}
```

Comment:
Why did you remove the fail fast option again?

Reply:
It crashed the workflow.. but that was a few commits ago. I can check it again (or, eh, google it :D)